modules/cnv-cdrom-wizard-fields-web.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_virtual_machines/cnv-create-vms.adoc
+// * cnv/cnv_vm_templates/cnv-creating-vm-template.adoc
+
+[id="cnv-cdrom-wizard-fields-web_{context}"]
+= CD-ROM fields
+
+|===
+|Media Type | Description
+
+|Container
+|Specify the container path. For example: kubevirt/fedora-registry-disk: latest.
+
+|Persistent volume claim
```

Comment:
It's "Attach Disk" here, not "Persistent volume claim".

There is a bug for it: https://bugzilla.redhat.com/show_bug.cgi?id=1813135

Reply:
yep indeed.

Reply:
Made the changes to the content to align with the notes provided in https://bugzilla.redhat.com/show_bug.cgi?id=1813135.